<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pagina_registro</class>
 <widget class="QMainWindow" name="Pagina_registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Registro&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre(s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_nombre">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_apellidos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>58</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_correo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>58</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Correo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar contraseña</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="registrar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrarme</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>¿Ya tienes una cuenta?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inicio_sesion">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar sesión</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tw Cen MT Condensed Extra Bold&quot;;color:rgb(170, 0, 0);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff5500;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
